tutorial/markdown/nodejs/express.md

```diff
@@ -22,343 +21,335 @@ sdk_language:
+│   ├── airportController.js
+│   └── routeController.js
+├── db
+│   ├── connection.js
+├── errors
+│   ├── errors.go
+├── src/routes
+│   ├── airline.js
+│   ├── airport.js
+│   ├── route.js
+├── src/shared
+│   ├── makeResponse.js
+│   ├── validateRequiredField.js
+├── src/app.js
+├── src/server.js
+├── Dockerfile
+└── __test__
 
-The import of the database connection uses [`/db/connection.js`](https://github.com/couchbase-examples/nodejs-quickstart/blob/master/db/connection.js):
 
-```js
-import * as couchbase from 'couchbase'
+```
 
-const options = { username: process.env.CB_USER, password: process.env.CB_PASS }
-const cluster = new couchbase.Cluster(process.env.CB_URL, options)
-const bucket = cluster.bucket(process.env.CB_BUCKET)
-const defaultScope = bucket.scope('_default')
-const profileCollection = defaultScope.collection('profile')
+We have separated out the  code into separate files by the entity (collection) in the `controllers` folder. The tests for the  project are present in the `__test__` folder.
 
-module.exports = { couchbase, cluster, profileCollection }
-```
+### Airport Entity
 
-In the [`connection.js`](https://github.com/couchbase-examples/nodejs-quickstart/blob/db/connection.js) file, we import the NodeJS SDK for Couchbase, create a connection to the database. The environment variables for `CB_URL`, `CB_USER`, `CB_PASS`, `CB_BUCKET`, which can be set for your development and testing environment respectively in the [`/config/` directory](https://github.com/couchbase-examples/nodejs-quickstart/blob/master/config). I have added both in the repo (`dev.env` and `test.env`), for the tutorial we will only focus on the dev environment. Feel free to explore the repo's [readme.md](https://github.com/couchbase-examples/nodejs-quickstart) file for additional instructions if you are interested in testing.
+For this tutorial, we will focus on the airport entity. The other entities are similar.
 
-After we have our configuration for database connection in place, we can create our express app:
+We will be setting up a REST API to manage airport documents.
 
-```js
-const app = express()
+- [POST Airport](#post-airport) – Create a new airport
+- [GET Airport](#get-airport) – Read specified airport
+- [PUT Airport](#put-airport) – Update specified airport
+- [DELETE Airport](#delete-airport) – Delete airport
+- [Airport List](#list-airport) – Get all airports. Optionally filter the list by country
+- [Direct Connections](#direct-connections) - Get a list of airports directly connected to the specified airport
 
-app.use(cors())
-app.use(express.json())
-app.use(express.urlencoded({ extended: true }))
-```
+For CRUD operations, we will use the [Node JS SDK](https://docs.couchbase.com/nodejs-sdk/current/hello-world/start-using-sdk.html) to create, read, update, and delete a document. Every document will need an ID (similar to a primary key in other databases) to save it to the database.
+
+## Document Structure
```

Comment:
For CRUD operations, we will use the KV operations in the Node JS SDK. The link should be https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html